sample_data_config/web-api/web-api-data-pipeline.config.yaml

```diff
@@ -20,21 +20,71 @@ webApi:
         - items
   #elifescience profile api
   - dataset: '{ENV}'
-    table: atest_profile_api
+    table: test_people_api
     dataUrl:
-      urlExcludingConfigurableParameters: https://api.elifesciences.org/profiles?
+      urlExcludingConfigurableParameters: https://api.elifesciences.org/people?
       configurableParameters:
         pageSizeParameterName: per-page
         defaultPageSize: 100
         pageParameterName: page
     response:
       itemsKeyFromResponseRoot:
         - items
+  # civi export
+  - dataset: '{ENV}'
+    table: civi_data_export
+    stateFile:
+      bucketName: '{ENV}-elife-data-pipeline'
+      objectName: 'airflow-config/generic-web-api/{ENV}-civi-export-processing-state.json'
+    urlSourceType:
+      name: 'civi'
+      sourceTypeSpecificValues:
+        fieldsToReturn:
+          - external_identifier
+          - tag
+          - group
+          - contact_type
+          - contact_sub_type
+          - last_name
+          - first_name
+          - middle_name
+          - email
+          - source
+          - user_unique_id
```

Comment:
There doesn't seem to be any entry where `user_unique_id` is populated (the `contact_id` and `id` field, which seem to be identical, are returned even if not listed)

Reply:
decision is not implementation decions, but a pipeline configuration decision, and yes, this should be there in the config

Reply:
I am looking at the fields to understand what they mean and what example values might mean. If we don't have a single example, then I think we can't be sure it's actually working or would work. (I moved this to a top level PR comment as it will be good to have a list of example values for some of the fields)